test/espnet2/enh/diffusion/test_score_based_diffusion.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+import torch
+from packaging.version import parse as V
+from torch import Tensor
+
+from espnet2.enh.diffusion.score_based_diffusion import ScoreModel
+
+is_torch_1_12_1_plus = V(torch.__version__) >= V("1.12.1")
```

Comment:
You don't need to add it because current ESPnet already dropped support for PyTorch versions before 1.12.1.

Reply:
removed.